packages/plugins/test/Plugin.test.ts

```diff
@@ -199,4 +199,38 @@ describe("Plugin", () => {
       expect(() => plugin.loadRecipe()).toThrow(expectedError);
     });
   });
+
+  describe("loadCompiler()", () => {
```

Comment:
It's great that you're adding tests for this functionality!

Reply:
Thx